components/InformationBar.tsx

```diff
@@ -110,6 +113,24 @@ export default function InformationDrawer(props: { requirementsDisplay: Requirem
     setValue(newValue);
   };
 
+  function getTotalCredits(s: SemesterType[]): any {
+    let total = 0;
+    completedCourses.forEach((c) => {
+      const courseSub: string = c.split("-")[0];
+      const courseNum: string = c.split("-")[1];
+      props.passedCourseList.forEach((pc) => {
+        if (pc.subject === courseSub && pc.number === courseNum) {
+          total += pc.credits;
```

Comment:
This counts a course every time it appears in the list...the problem with this is that a course is in the list once for each category it is in. For example, use ANTH-220 as a completed course and you will get 9 out of 120 credits because it is worth 3 credits and in 3 categories. Try using .find on the passedCourseList instead of forEach.

Reply:
I came up with a different solution that is a bit more "hacky" I guess, but it works